spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaProperties.java

```diff
@@ -93,6 +94,8 @@
 
 	private final Security security = new Security();
 
+	private final RetryTopic retryTopic = new RetryTopic();
```

Comment:
This would create a `retry-topic` group and we don't use dashes in group parts. Perhaps a `Retry` inner type with a `Topic` if that makes sense to separate them. This would give `spring.kafka.retry.topic`.